browser/src/control/Control.UIManager.js

```diff
@@ -711,6 +711,85 @@ L.Control.UIManager = L.Control.extend({
 		this.map.fire('hidebusy');
 	},
 
+	showHyperlinkWarn: function(url, action) {
+		console.debug(action);
+		var json = {
+			id: 'hyperlinkWarnDialog',
+			dialogid: 'hyperlinkWarnDialog',
+			type: 'dialog',
+			text: 'Are you sure you want to visit the following URL?',
+			title: 'Hyperlink',
+			jsontype: 'dialog',
+			enabled: true,
+			children: [
```

Comment:
`responses` group was removed because we only use that when core expects an answer. Example in the repair document dialog we do need that see `Control.DocumentRepair.js` and the respective 
```javascript
			responses: [
				{
					id: 'ok',
					response: 1
				},
				{
					id: 'cancel',
					response: 0
				},
			],
```